tests/expected/intrinsics/write_bytes/out-of-bounds/expected

```diff
@@ -0,0 +1,2 @@
+FAILURE\
+memset destination region writeable
```

Comment:
newline?

Reply:
I don't include newlines in these because the `expected` testing suite is space-sensitive: `compiletest` would actually check if there's an empty line in the output if we add it.